pkg/ipamd/ipamd.go

```diff
@@ -1147,9 +1252,36 @@ func GetConfigForDebug() map[string]interface{} {
 		envWarmIPTarget:     getWarmIPTarget(),
 		envWarmENITarget:    getWarmENITarget(),
 		envCustomNetworkCfg: UseCustomNetworkCfg(),
+		envAssignIPv4:       getAssignIPv4(),
+		envAssignIPv6:       getAssignIPv6(),
+	}
+}
+
+func getEnvBool(name string, def bool) bool {
```

Comment:
There is a `getBoolEnvVar(name string, defaultValue bool)` in network.go already, maybe we should make some kind of utility function for this?